instance/static/html/instance/index.html

```diff
@@ -63,7 +68,13 @@
                                                ng-click="provision(selected.instance)">
               Reprovision
             </button>
-            <img src="/static/img/running.png" tooltip-placement="bottom" tooltip="Provisioning in progress..." alt="Provisioning in progress..." ng-switch-default />
+            <button class="button reprovision" ng-switch-when="provisioning_failure"
+                                               ng-click="provision(selected.instance)">
+              Reprovision
+            </button>
+            <i class="fa fa-2x fa-ellipsis-h" tooltip-placement="right" tooltip="Provisioning in progress" ng-switch-default></i>
+            <i class="fa fa-2x fa-times" tooltip-placement="right" tooltip="Provisioning failure" ng-switch-when="provisioning_failure"></i>
+            <i class="fa fa-2x fa-check" tooltip-placement="right" tooltip="Ready" ng-switch-when="ready"></i>
```

Comment:
This duplicates https://github.com/open-craft/opencraft/pull/13/files#diff-a8e64e20aeb5487bab76b72ae22e9aefR28 . The same comment applies, plus refactoring the test within the client, to only define it once - this is worth adding as a method to the controller.